api/db/populate-attendants.js

```diff
@@ -100,6 +104,10 @@ const createAttendants = async (verbose) => {
   }
 
   console.log('\x1b[32m', `Created ${attendantsArray.length} attendants`);
+  console.log(
+    '\x1b[32m',
+    `Created ${notificationsCount} attendant notifications`
+  );
```

Comment:
Maybe, to avoid too much output, all of this could be merged in one line, something like:

```javascript
console.log(
  '\x1b[32m',
  `Created ${attendantsArray.length} attendants (and ${notificationsCount} related notifications)`
);

```
I don't feel strongly about this though, so feel free to leave it as is!